.github/workflows/release-nightly.yml

```diff
@@ -0,0 +1,31 @@
+name: Realease nightly
+
+on:
+  workflow_dispatch:
```

Comment:
@vmaerten @pd93 Good to have in mind the with `workflow_dispatch` we can trigger this manually after doing an important fix.

Useful when to ask users to try nightly without waiting for the next day.